index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Welcome Page</title>
+</head>
+<body>
+	<h1>Welcome Page</h1>
+	<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptas, ab tempore quia autem praesentium dolorem dolore, veritatis numquam maiores debitis odio ullam, doloremque omnis dignissimos assumenda ratione repudiandae, dolorum cum.</p>
```

Comment:
Change it to a meaningful text.

Reply:
THis text is just a temporary text. We will change it later

Reply:
OKay then